code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -65,19 +75,27 @@
 	name = "\improper Mysterious Medibot"
 	desc = "International Medibot of mystery."
 	skin = "bezerk"
-	treatment_oxy = "perfluorodecalin"
-	treatment_brute = "styptic_powder"
-	treatment_fire = "silver_sulfadiazine"
-	treatment_tox = "charcoal"
+	treatments = list(
```

Comment:
Reee, why do you suddenly change the order of the list from the order in the initial declaration of the variable?

Please standardise these for my sanity.

Reply:
Done